cmd/grpc-health-probe/main.go

```diff
@@ -3,11 +3,12 @@ package main
 import (
 	"context"
 	"flag"
-	"log"
 	"os"
 	"os/signal"
 	"time"
 
+	"github.com/aws/amazon-vpc-cni-k8s/pkg/utils/logger"
+
```

Comment:
I think it's fine that the gRPC health check is just using the standard go library. These logs are not essential. @jaypipes, any comments on this?

Reply:
meh, it's fine to leave it here I guess.